src/foam/nanos/auth/CountryService.java

```diff
@@ -72,7 +70,7 @@ protected void notifySupport(String query) {
     emailMessage.setSubject ("Unknown Country");
     emailMessage.setBody    ("User just added an unknown country " + query);
     emailMessage.setTo(new String[]{"ops@nanopay.net"});
-    emailService.sendEmail(getX(), emailMessage);
+    EmailsUtility.sendEmailFromTemplate(getX(), null, emailMessage, null, null);
```

Comment:
Sorry I didn't notice that we were using this to send non-templated emails. I think this design still needs a little work but at least we're in a better state than before so let's roll with this given the current time crunch. Please file a task to revisit this and remove the use of this utility class in favour of a service that knows how to send emails given a template. We'll have to think about that interface a little but we'll tackle that later.

Reply:
https://nanopay.atlassian.net/browse/CPF-1290